sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/FilterTest.java

```diff
@@ -61,125 +60,109 @@ public Boolean apply(Integer elem) {
     p.run();
   }
 
-  @Deprecated
   @Test
   @Category(RunnableOnService.class)
-  public void testFilterBy() {
+  public void testFilterByPredicate() {
     TestPipeline p = TestPipeline.create();
 
     PCollection<Integer> output = p
         .apply(Create.of(1, 2, 3, 4, 5, 6, 7))
-        .apply(Filter.by(new EvenFn()));
+        .apply(Filter.byPredicate(new EvenFn()));
 
     PAssert.that(output).containsInAnyOrder(2, 4, 6);
     p.run();
   }
 
   @Test
   @Category(RunnableOnService.class)
-  public void testIdentityFilterByPredicate() {
-    TestPipeline p = TestPipeline.create();
-
-    PCollection<Integer> output = p
-        .apply(Create.of(591, 11789, 1257, 24578, 24799, 307))
-        .apply(Filter.byPredicate(new TrivialFn(true)));
-
-    PAssert.that(output).containsInAnyOrder(591, 11789, 1257, 24578, 24799, 307);
-    p.run();
-  }
-
-  @Test
-  @Category(RunnableOnService.class)
-  public void testNoFilterByPredicate() {
+  public void testFilterLessThan() {
     TestPipeline p = TestPipeline.create();
 
     PCollection<Integer> output = p
-        .apply(Create.of(1, 2, 4, 5))
-        .apply(Filter.byPredicate(new TrivialFn(false)));
+        .apply(Create.of(1, 2, 3, 4, 5, 6, 7))
+        .apply(Filter.lessThan(4));
 
-    PAssert.that(output).empty();
+    PAssert.that(output).containsInAnyOrder(1, 2, 3);
     p.run();
   }
 
   @Test
   @Category(RunnableOnService.class)
-  public void testFilterByPredicate() {
+  public void testFilterGreaterThan() {
     TestPipeline p = TestPipeline.create();
 
     PCollection<Integer> output = p
         .apply(Create.of(1, 2, 3, 4, 5, 6, 7))
-        .apply(Filter.byPredicate(new EvenFn()));
+        .apply(Filter.greaterThan(4));
 
-    PAssert.that(output).containsInAnyOrder(2, 4, 6);
+    PAssert.that(output).containsInAnyOrder(5, 6, 7);
     p.run();
   }
 
   @Test
   @Category(RunnableOnService.class)
-  public void testFilterLessThan() {
+  public void testFilterLessThanEq() {
     TestPipeline p = TestPipeline.create();
 
     PCollection<Integer> output = p
         .apply(Create.of(1, 2, 3, 4, 5, 6, 7))
-        .apply(Filter.lessThan(4));
+        .apply(Filter.lessThanEq(4));
 
-    PAssert.that(output).containsInAnyOrder(1, 2, 3);
+    PAssert.that(output).containsInAnyOrder(1, 2, 3, 4);
     p.run();
   }
 
   @Test
   @Category(RunnableOnService.class)
-  public void testFilterGreaterThan() {
+  public void testFilterGreaterThanEq() {
     TestPipeline p = TestPipeline.create();
 
     PCollection<Integer> output = p
         .apply(Create.of(1, 2, 3, 4, 5, 6, 7))
-        .apply(Filter.greaterThan(4));
+        .apply(Filter.greaterThanEq(4));
 
-    PAssert.that(output).containsInAnyOrder(5, 6, 7);
+    PAssert.that(output).containsInAnyOrder(4, 5, 6, 7);
     p.run();
   }
 
   @Test
   public void testDisplayData() {
-    ParDo.Bound<Integer, Integer> lessThan = Filter.lessThan(123);
+    Filter<Integer> lessThan = Filter.lessThan(123);
```

Comment:
Style nit: Probably don't even need to assign this to a variable. It is already quite readable inlined.